src/utils/generateDirectionalStyles.js

```diff
@@ -2,11 +2,13 @@ import rtlCSSJS from 'rtl-css-js';
 
 import { LTR_SELECTOR, RTL_SELECTOR } from './withRTLExtension';
 
+const directionRegex = /Left|Right/;
```

Comment:
Will this always be at the end of the string? If so, it might be nice to anchor this regex at the end.

Reply:
Yes, I think you're right.